helpers.go

```diff
@@ -93,11 +93,24 @@ func ValidateTargetTable(table *Table) (*ValidatedTable, error) {
 		return nil, fmt.Errorf("table is not target")
 	}
 
-	if !tableNameRegEx.MatchString(table.String()) {
-		return nil, &ErrTableNameWrongFormat{Name: table.String()}
+	closingChar := map[byte]byte{
+		'"': '"',
+		'`': '`',
+		'[': ']',
 	}
 
-	parts := strings.Split(table.String(), "_")
+	tableName := table.String()
+	for start, end := range closingChar {
+		if tableName[0] == start && tableName[len(tableName)-1] == end {
+			tableName = tableName[1 : len(tableName)-1]
```

Comment:
remove the enclosing chars for validation

Reply:
looks good